deleteCognitoUsers.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# A simple script to delete all users from a cognito user pool. NOTE: jq must be installed
+
+COGNITO_USER_POOL_ID=your-user-pool-id
```

Comment:
This is just like what I was suggesting with the configurable consts up near the top of the file.

Reply:
Implemented